submissions/eve5ince/html-css-popup/css/master.css

```diff
@@ -0,0 +1,141 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	text-decoration: none;
+	list-style-type: none;
+	color: #757575;
+}
+body {
+	background: rgb(238,210,232);
+	background: linear-gradient(45deg, rgba(238,210,232,0.7889355571330094) 22%, rgba(99,252,69,0) 100%); 
+	font-family: Arial, Helvetica, sans-serif;
+}
+nav>ul {
+   display: flex;
+   justify-content: space-between;
+	margin-top: 30px;
+	margin-left: 900px;
+	align-items: center;
+	font-size: 18px;
+	background-color: transparent;
+	padding: 5px 0 5px 0;
+	border-radius: 25px;
+	letter-spacing: 0.05em;
+}
+nav>ul>li>a:hover{
+	transition-duration: 0.22s;
+	opacity: 0.7;
+}
+.nav-img {
+	display: block;
+	background-repeat: no-repeat;
+	background-size: contain;
+	width: 32px;
+	height: 32px;
+}
+.control {
+	display: none;
+	position: absolute;
+	top: 100px;
+	right: 96px;
+	width: 320px;
+	height: 460px;
+	border: 2px groove;
+	background: transparent;
+	border: 1px solid;
+	overflow: auto;
+	border-radius: 22px;
+
+	-webkit-scrollbar: thin;
+	-webkit-scrollbar: #f9dcc4 transparent;
+}
+::-webkit-scrollbar {
+    width: 12px;
+}
+
+::-webkit-scrollbar-track {
+    -webkit-box-shadow: inset 0 0 4px rgba(0,0,0,0.3); 
+    border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    -webkit-box-shadow: inset 0 0 4px rgba(0,0,0,0.5); 
+}
+.links, .more {
+	display: grid;
+   grid-template-columns: repeat(3, 1fr);
+	font-size: 12px;	
+	text-align: center;
+	grid-gap: 10px;
+	padding: 0px 10px 0px 10px;
+}
+.links>a {
+	padding-bottom: 5px;
+}
+.links>a:hover, .more>a:hover{
+	outline: 1px solid silver;
+}
+.links>a:focus, .more>a:focus {
+	 background: rgba(238,210,232, 0.2); 
+}
+.links>img>p {
+	font-size: 14px;
+}
+li:first-of-type {
```

Comment:
Try `li + li { padding: 0 15px; }` instead `li:first-of-type` and `li:last-of-type`

Reply:
Thx, I ended simply with adding margins.